lib/presentation/screens/auth/signin.dart

```diff
@@ -2,34 +2,140 @@ import 'package:flutter/material.dart';
+                  const EdgeInsets.symmetric(vertical: 120, horizontal: 24),
+              child: Container(
+                constraints: const BoxConstraints(maxWidth: 400),
+                decoration: BoxDecoration(
+                  color: HexColor("0F1012").withOpacity(0.35),
+                  border: Border.all(
+                    color: Colors.white,
+                    width: 2,
+                  ),
+                  borderRadius: BorderRadius.circular(12),
+                ),
+                child: Padding(
+                  padding: const EdgeInsets.all(16),
+                  child: Center(
+                    child: Column(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        crossAxisAlignment: CrossAxisAlignment.center,
+                        children: [
+                          const Padding(
+                            padding: EdgeInsets.only(top: 8, bottom: 24),
+                            child: Text(
+                              'SIGN IN',
+                              style: TextStyle(
+                                  fontSize: 20, fontFamily: 'Poppins'),
+                            ),
+                          ),
+                          _SignInForm(
+                            emailController: _emailController,
+                            passwordController: _passwordController,
+                            obscurePassword: _obscurePassword,
+                          ),
+                          const SizedBox(height: 16),
+                          button(label: "Login", onPressed: _handleLogin),
+                          const SizedBox(
+                            height: 16,
+                          ),
+                          const Row(
+                            children: [
+                              Expanded(
+                                child: Divider(
+                                  thickness: 1,
+                                ),
+                              ),
+                              Padding(
+                                padding: EdgeInsets.symmetric(horizontal: 8.0),
+                                child: Text(
+                                  "or",
+                                  style: TextStyle(fontFamily: 'Poppins'),
+                                ),
+                              ),
+                              Expanded(
+                                child: Divider(
+                                  thickness: 1,
+                                ),
+                              ),
+                            ],
+                          ),
+                          const SizedBox(
+                            height: 16,
+                          ),
+                          button(label: 'Register', onPressed: _handleRegister),
+                          const SizedBox(
+                            height: 24,
+                          ),
+                        ]),
```

Comment:
ini lagi masih numpuk!!!!!!!!!!!!!!!!!
```suggestion
                        ],),
```